src/render-dom.js

```diff
@@ -154,11 +154,11 @@ function makeDiffAndPatchToElement$(rootElem) {
 function renderRawRootElem$(vtree$, domContainer, {CERegistry, driverName}) {
   let diffAndPatchToElement$ = makeDiffAndPatchToElement$(domContainer)
   return vtree$
-    .flatMapLatest(transposeVTree)
+    .switchMap(transposeVTree)
     .startWith(VDOM.parse(domContainer))
     .map(makeReplaceCustomElementsWithWidgets(CERegistry, driverName))
-    .doOnNext(checkRootVTreeNotCustomElement)
-    .pairwise()
+    .do(checkRootVTreeNotCustomElement)
+    .bufferCount(2)
```

Comment:
Then that is new. I checked with Rx 4, and `bufferCount(2)` gave me the same as `pairwise()`.